<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1845</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="adapterListLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>本地适配器</string>
    </property>
   </widget>
   <widget class="QComboBox" name="adapterList">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>901</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="packTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>1141</width>
      <height>721</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="packInfo">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>300</y>
      <width>551</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>50</y>
      <width>551</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>过滤器</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="4">
      <widget class="QCheckBox" name="ARPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ARP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="filterLabel_6">
       <property name="text">
        <string>IP过滤</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QCheckBox" name="orPortCheck">
       <property name="text">
        <string>或</string>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QCheckBox" name="orIPCheck">
       <property name="text">
        <string>或</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="IPv4Check">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>IPv4</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="7">
      <widget class="QLineEdit" name="dIPFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="TCPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>TCP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="elseCheckL2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>网络层其他</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QCheckBox" name="sIPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>源IP</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="UDPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>UDP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="IPv6Check">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>IPv6</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="filterLabel_5">
       <property name="text">
        <string>端口号过滤</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="filterLabel_3">
       <property name="text">
        <string>网络层</string>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QCheckBox" name="dIPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>目的IP</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="filterLabel_4">
       <property name="text">
        <string>传输层</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QSpinBox" name="sPortFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="12" column="7">
      <widget class="QSpinBox" name="dPortFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QCheckBox" name="sPortCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>源端口</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="5">
      <widget class="QCheckBox" name="dPortCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>目的端口</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLineEdit" name="sIPFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="ICMPCheck">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ICMP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="3" colspan="3">
      <widget class="QPushButton" name="startButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="filterLabel_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>adapterList</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>devSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>1050</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>packTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>packSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sPortCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sPortFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1345</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>1435</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dIPCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dIPFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1614</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>1683</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dPortCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dPortFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1608</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>1667</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sIPCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sIPFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1341</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>1436</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1434</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>1122</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearPack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>devSelected(int)</slot>
  <slot>packSelected(QModelIndex)</slot>
  <slot>filter()</slot>
  <slot>clearPack()</slot>
 </slots>
</ui>
